### DISCUSSION TITLE
Quasar 0.17 Modal - When a modal is shown, can I interactive with the background without closing the modal?

### DISCUSSION THREAD

**Discussion by matteo1222** [quasarframework/quasar#9890] [General - Components / Directives / etc]
I would like to interact with the modal backdrop without closing the modal. Is there any way to achieve this? Thank you in advance!

**Comment by MilosPaunovic** [comment]
You can use `no-backdrop-dismiss` prop, as shown [here](https://v0-17.quasar-framework.org/components/modal.html#Vue-Properties).

**Reply by matteo1222** [reply]
Thank you for the reply! I think I didn't explain my attempt very clear. What I am trying to do is to interact with elements other than the modal (while keeping the modal).
Only using the no-backdrop-dismiss forces me to only interact with the modal.
I found a workaround by setting the visibility (through CSS) of modal-content's parent node to hidden.
And set modal-content to visibility: visible.

**Comment by ontwikkelfabriek** [comment]
@matteo1222 

Just very curious, is Quasar v0.X still viable to use in 2021? Are you using SPA mode?

**Reply by matteo1222** [reply]
We are using SPA mode, and I think it works fine because it caters to most of our needs. We started using 0.17 years ago, so we are worried that upgrading might break things.
